airbyte-integrations/connectors/source-toggl/source_toggl/toggl.yaml

```diff
@@ -0,0 +1,112 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  data_selector:
+    extractor:
+      field_pointer: ["data"]
+  requester:
+    url_base: "https://api.track.toggl.com"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        start_date: "{{ config['start_date'] }}"
+        end_date: "{{ config['end_date'] }}"
```

Comment:
What happen with null start date and end date?

Reply:
The start and end date are now required.